### PR TITLE
An extremely lazy copy paste of Cit's animal sound emotes.

### PR DESCRIPTION
**Pull request by BlackMajor**
Y'all can *weh and *bark now etc.

~~Also remind me to strangle whoever the hell coded this in the first place why is there so much spaghetti just to make sure people don't emote spam.~~
Spaghetti removed

~~If somebody who's responsible for code stuff on virgo tells me to just slap a cooldown on emote spamming I'll do it cause that's easier than this meme junk.~~
Cooldown added

Full list of added emotes are

*awoo (now featuring sound)
*nya
*hiss
*bark
*merp
*weh
*chirp
*peep

Edit: Pending change to simple cooldown instead of big die danger.... probably tomorrow, I'm sleepy.

Edit 2: Changes made, global emote cooldown set at 1.2seconds

### CODE REVIEW

**Comment by KasparoVy** [comment]
Fantastic, thanks much.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -20,6 +20,9 @@
 	var/active_regen = FALSE //Used for the regenerate proc in human_powers.dm
 	var/active_regen_delay = 300
 
+	var/emoteDanger = 1 // What the current danger for spamming emotes is - shared between different types of emotes to keep people from just
+										// flip/snap/flip/snap.  Decays at a rate of 1 per second to a minimum of 1.
+
```
Comment (Novacat) [inline code/modules/mob/living/carbon/human/human.dm]: This should be moved to human_vr.dm and added to the dmi.

File: code/modules/mob/living/carbon/human/life.dm [hunk 2]
```diff
@@ -37,6 +37,8 @@
 	set invisibility = 0
 	set background = BACKGROUND_ENABLED
 
+	var/timeSinceLastTick = world.time - lastLifeProc //Used to curb emote spam
+
```
Comment (Novacat) [inline code/modules/mob/living/carbon/human/life.dm]: Moved to life_vr.dm as well.

File: code/modules/mob/living/carbon/human/life.dm [hunk 3]
```diff
@@ -95,6 +97,10 @@
 
 	pulse = handle_pulse()
 
+	//Used to determine if someone should get ass blasted for awoo'ing
+	emoteDanger = max(0, emoteDanger - (timeSinceLastTick / 10))
```
Comment (Novacat) [inline code/modules/mob/living/carbon/human/life.dm]: //VOREStation Edit

File: code/modules/mob/living/living_defines.dm [hunk 4]
```diff
@@ -68,4 +68,6 @@
 
 	var/looking_elsewhere = FALSE //If the mob's view has been relocated to somewhere else, like via a camera or with binocs
 
-	var/image/selected_image = null // Used for buildmode AI control stuff.
\ No newline at end of file
+	var/image/selected_image = null // Used for buildmode AI control stuff.
+
+	var/lastLifeProc = 0    // Wallclock time of the last time this mob had life() called on them (at finish), used for tying variables to wallclock rather than tickcount
```
Comment (Novacat) [inline code/modules/mob/living/living_defines.dm]: Put this in a vr file as well.

**Comment by ArgobargSoup** [comment]
...okay... if... if I'm reading the code right... if the danger stuff triggers... you have a chance of ***having your head fall off/asplode*** from awooing.

This is *absolutely hilarious*, but... also seems a bit much.  \:p

**Comment by KasparoVy** [comment]
The toned down version would be imploding your lungs/oxyloss or whatever

**Comment by Screemonster** [comment]
Yeah, I'd rather have zero chance if it's not been done for a while.

**Comment by ArgobargSoup** [comment]
I mean, sure, flipping can break legs...  the thing is, for synths?  It's coded so that where organics would have limbs break, synths just have the part fall off... which is hilarious, to be honest, and at least putting synth limbs back on isn't all that hard.

But with these new ones, where the involved parts that can fall off include the *head*...  As equally hilarious as it is to have a head just fall off, it also kinda *kills* the synth, since from what I remember, synths still can't survive without a head.  \:P

**Comment by BlackMajor** [comment]
Yeah I'm just gonna change it so these emotes just have a simple cooldown attached rather than meme code.

It's funny like, once or twice, but memes get old.

**Comment by TheDavestDave** [comment]
I take it the emote x9001 is harder to code than it looks. Hug X 100 would be useful to a lot of people

**Comment by KasparoVy** [comment]
Excellent!

**Pull request merged**